src/render/NewGameScreen.java

```diff
@@ -11,6 +11,11 @@
 public class NewGameScreen extends Screen {
     private ArrayList<GuiElement> left = new ArrayList<>();
     private ArrayList<GuiElement> right = new ArrayList<>();
+    private ToggleButton color1Button;
+    private ToggleButton color2Button;
+    private ToggleButton color3Button;
+    private ToggleButton color4Button;
+    private boolean atLeastOneSelected;
```

Comment:
This doesn't need to be a class field. It can be calculated at any time from the buttons.

Reply:
atLeastOneSelected removed as a class field.